DynamicAnalyzer/views/android/environment.py

```diff
@@ -46,10 +46,28 @@ def connect_n_mount(self):
         self.wait(2)
         logger.info('Connecting to Android %s', self.identifier)
         out = subprocess.check_output([get_adb(), 'connect', self.identifier])
+        self.wait(2)
         if b'unable to connect' in out or b'failed to connect' in out:
             logger.error('%s', out.decode('utf-8').replace('\n', ''))
             return False
         else:
+            # TODO: proof if not already connected as admin 
+            # and not already FS is mounted
+            # start adb as root
+            logger.info('Restarting ADB Daemon as root')
+            out = subprocess.check_output([get_adb(), 'root'])
```

Comment:
Genymotion by default has root user in shell. Why is this required?

Reply:
Yes, you are right - but in case somebody is using another "device" it is required and since it does not has any negative effect on genymotion it would be nice to eventually support other setups. (I am hosting my whole setup as a docker-compose script - if you are interested in, I could share it, after I finis my studies with your nice tool!)

Reply:
That would be interesting. We were testing docker support, the issues we faced were getting proxy to work and some adb conflict if there is an adb server running in host.

Reply:
I created a gist for you to have a look, as it is my actual setup (running on a Kali Linux VM on VMware Workstation 15 Pro on a Windows 10 host system). I you have further questions we could get in contact! :)
See: https://gist.github.com/toxicphreAK/8ecfaecb76c69900e1bb95f52324d5af

Reply:
This looks interesting. I will definitely take a look after my vacation.